Database/Corrections/sodItemFixes.lua

```diff
@@ -1,4 +1,4 @@
----@type SeasonOfDiscovery
+---@class SeasonOfDiscovery
```

Comment:
I still think it's wrong to have multiple `class` declarations of the same class. I do get why you changed it this way though.